### PR TITLE
mimxrt: Add and change build scripts for nightly builds.

### PR DESCRIPTION
**Pull request by robert-hh**
This scripts create the  files for Teensy 4.0 and Teensy 4.1. 
User of MIMXRT10xx_EVK board should be able do build the firmware
themselves. They might need specific DEBUG settings.

### CODE REVIEW

File: tools/autobuild/build-mimxrt-latest.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+# check we are in the correct directory
+if [ ! -r modmimxrt.c ]; then
+    echo "must be in mimxrt directory"
+    exit 1
+fi
+
+# build the boards
+do_build Teensy_4.0 TEENSY40 hex
+do_build Teensy_4.1 TEENSY41 hex
```
Comment (dpgeorge) [inline tools/autobuild/build-mimxrt-latest.sh:35]: I think it's better to have the name of the file match the board-dir name, and also not have "." in it (to not confuse Windows).

Comment (robert-hh) [reply tools/autobuild/build-mimxrt-latest.sh]: Thanks. Changed.

**Comment by dpgeorge** [comment]
Thanks for updating, merged in efa97beb992049978f0e7be6e4e2f01249054362

**Comment by robert-hh** [comment]
Hello Damien, I had expected to see the mimxrt builds in the daily builds now, but they are not. Is something failing or is the changed script not yet activated?

**Comment by robert-hh** [comment]
Today, the MIMXRT1020_EVK appeared on the daily build list, but not the Teensy builds, even if they should be created with the same script.

**Comment by dpgeorge** [comment]
I only just activated this new script (it was previously using the private version, before committing it to this repo).

The Teensy builds are there, eg https://micropython.org/resources/firmware/TEENSY40-20210705-unstable-v1.16-56-g031fe0f14.hex

They are not yet in any list because I didn't yet make an mimxrt section on the download page.

**Comment by robert-hh** [comment]
Thanks. I looked at the page https://micropython.org/download/all/, where I had expected to find these, even if a MIMXRT section does not exist. But there is only the MIMXRT1020_EVK file.

**Comment by dpgeorge** [comment]
Ah, I forgot about the download/all page.  That page doesn't list `.hex` files so that's why Teensy is missing.  But it's now fixed, and is now there.

**Comment by robert-hh** [comment]
Thanks. That's what I suspected.

**Pull request closed without merging**